### ISSUE TITLE
UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead

### ISSUE THREAD

**Issue by rachmadvwp** [BindsNET/bindsnet#357]
Hi,
I tried to run BindsNET in my other machines, but I got this warning and no printout of accuracy
`/opt/conda/conda-bld/pytorch_1579022027550/work/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.`
Do you know why this happens and how can we solve this? 
Thanks in advance.

**Comment by Hananel-Hazan** [comment]
Thank you for using BindsNET. 

Please provide more information how the error above can be replicated. The error above is not clear where the problem originate, it could be a cuda issue too. 

Also, its seems that you using conda. Try to uninstall BindsNET from pip and install the conda package 
` conda install -c bindsnet bindsnet `

**Comment by rachmadvwp** [comment]
Hi @Hananel-Hazan ,
Will this install can be modified without reinstalling it again like `pip install -e .` ?
Because I need to modify it afterward for implementing my technique. Thanks in advance.

**Comment by Hananel-Hazan** [comment]
Hi @rachmadvwp , you are right, the conda install method will install BindsNET in production/release mode, not development mode like in `pip install -e .` 

Can you run tests examples with using only PyTorch?
My suspicion is that the problem is with the Cuda/PyTorch installation that BindsNET heavy rely on.

**Comment by rachmadvwp** [comment]
Hi @Hananel-Hazan 
Here is my `pip list`. Does it help?

Package         Version                           
--------------- ------------------- 
attrs           19.3.0              
bcrypt          3.1.7               
bindsnet        0.2.3               
certifi         2019.11.28          
cffi            1.14.0              
cloudpickle     1.3.0               
cryptography    2.8                 
cycler          0.10.0              
Cython          0.29.15             
decorator       4.4.2               
experiments     0.1                 
future          0.18.2              
gym             0.17.1              
imageio         2.8.0               
joblib          0.14.1              
kiwisolver      1.1.0               
matplotlib      3.2.0               
mkl-fft         1.0.15              
mkl-random      1.1.0               
mkl-service     2.3.0               
more-itertools  8.2.0               
networkx        2.4                 
numpy           1.18.1              
olefile         0.46                
opencv-python   4.2.0.32            
packaging       20.3                
pandas          1.0.1               
paramiko        2.7.1               
Pillow          7.0.0               
pip             20.0.2              
pluggy          0.13.1              
py              1.8.1               
pycparser       2.20                
pyglet          1.5.0               
PyNaCl          1.3.0               
pyparsing       2.4.6               
pytest          5.3.5               
python-dateutil 2.8.1               
pytz            2019.3              
PyWavelets      1.1.1               
PyYAML          5.3                 
scikit-image    0.16.2              
scikit-learn    0.22.2.post1        
scipy           1.4.1               
setuptools      45.2.0.post20200210 
six             1.14.0              
torch           1.4.0               
torchvision     0.5.0               
tqdm            4.43.0              
wcwidth         0.1.8               
wheel           0.34.2

**Comment by Hananel-Hazan** [comment]
Yes, it help. But still not sure why you have that error. 

1. When you run it on CPU does it run without issues?

2.  Can you try to run [this](https://github.com/pytorch/examples/blob/master/mnist/main.py) example? using your GPU.
